zilliqa/src/consensus.rs

```diff
@@ -1029,7 +1026,7 @@ impl Consensus {
 
         committee.add_validators(self.pending_peers.drain(..));
 
-        if committee.len() <= 1 {
+        if false {
```

Comment:
will revert for merge to main

Reply:
I'd be in favour of just removing this `if` block. There's no reason a network with 1 validator shouldn't work fine (in fact it does, with this log message being the only annoyance).

Reply:
It is only useful in a dev environment, although I think it's impossible currently to start the network as more than a single node :)

I'd approve a PR if you want to add that flag :)